app/views/tag/blog.html.erb

```diff
@@ -25,7 +25,7 @@
         <% end %>
       <% else %>
         <div class="alert alert-info" role="alert">
-        <a href="/login?return_to=/blog" class="alert-link">Log in</a> to get updates for new posts!
+        <a data-toggle="modal" data-target="#loginModal?return_to=/blog">Log in</a> to get updates for new posts!
         </div>
```

Comment:
so would I add this:
@SidharthBansal 
```
 <% if current_user.following(params[:id]) %>
         <a rel="tooltip" title="<%= t('tag.blog.unfollow') %>" class="btn btn-default btn-sm active" href="/unsubscribe/tag/<%= params[:id] %>"><i class="fa fa-eye"></i> <%= t('tag.blog.following') %> <b><%= params[:id] %></b></a>
```
here? How would I make it so they unfollow when done login in? I know that the above handles  the unfollow, I just don't know how I would link it to here